db/migrate/20210929132645_core_setting_to_dsl.rb

```diff
@@ -0,0 +1,9 @@
+class CoreSettingToDsl < ActiveRecord::Migration[6.0]
+  def up
+    Setting.where(category: 'Setting::Auth').update_all(category: 'Setting')
+    Setting.where(category: 'Setting::Email').update_all(category: 'Setting')
+    Setting.where(category: 'Setting::Notification').update_all(category: 'Setting')
+    Setting.where(category: 'Setting::Provisioning').update_all(category: 'Setting')
+    Setting.where(category: 'Setting::Puppet').update_all(category: 'Setting')
```

Comment:
Feels like you can make a list of old category names and have a single `Setting.where().update_all()` them all at the same time.